README.md

```diff
@@ -1,68 +1,132 @@
+<p align="center">
+  <a href="https://uploadcare.com/?ref=github-readme">
+    <picture>
+      <source media="(prefers-color-scheme: dark)" srcset="./assets/media/logo-safespace-transparent.svg">
```

Comment:
Let's use inverted one for the dark mode
<img width="314" alt="logo-transparent-inverted@2x" src="https://github.com/uploadcare/blocks/assets/670959/336c03fa-1937-4745-b685-7a7e55306d82">

Reply:
Just mixed up the order of assets. Changed!